src/puyapy-stubs/arc4.pyi

```diff
@@ -350,7 +350,14 @@ class DynamicArray(_ABIEncoded, typing.Generic[_TArrayItem], Reversible[_TArrayI
 class Address(StaticArray[Byte, typing.Literal[32]]):
     """An alias for an array containing 32 bytes representing an Algorand address"""
 
-    def __init__(self, account_or_bytes: puyapy.Bytes | puyapy.Account | bytes, /): ...
+    def __init__(self, value: puyapy.Account | str | puyapy.Bytes = ..., /):
```

Comment:
No `bytes` literal anymore?

Reply:
It didn't make sense here, the address `str` is much more usable.
I'll add `bytes` literals to `from_bytes`.